docs/src/content/docs/tr/guides/authoring-content.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Markdown'da İçerik Yazmak
+description: Starlight'ın desteklediği Markdown sözdizimine genel bakış.
+---
+
+Starlight, `.md` uzantılı dosyalarda [Markdown](https://daringfireball.net/projects/markdown/) sözdizimini, [YAML](https://dev.to/paulasantamaria/introduction-to-yaml-125f) önbölümünde olduğu gibi başlık ve açıklama gibi metaverileri tanımlamak için destekler.
+
+Markdown desteği ve kullanımı farklılık gösterebileceğinden dolayı, bu dosya formatlarını kullanıyorsanız [MDX dokümantasyonu](https://mdxjs.com/docs/what-is-mdx/#markdown) ya da [Markdoc dokümantasyonu](https://markdoc.dev/docs/syntax)'nu incelediğinizden emin olun.
+
+## Satır İçi Stiller
+
+Metin **kalın**, _italik_ ya da ~~üstü çizili~~ olabilir.
+Text can be **bold**, _italic_, or ~~strikethrough~~.
+
+```md
+Metin **kalın**, _italik_ ya da ~~üstü çizili~~ olabilir.
+```
+
+Başka bir sayfaya [bağlantı ekleyebilirsiniz](/getting-started/).
+
+```md
+Başka bir sayfaya [bağlantı ekleyebilirsiniz](/getting-started/).
```

Comment:
Links should point to the Turkish versions here:

````suggestion
Başka bir sayfaya [bağlantı ekleyebilirsiniz](/tr/getting-started/).

```md
Başka bir sayfaya [bağlantı ekleyebilirsiniz](/tr/getting-started/).
````